scala/project/Build.scala

```diff
@@ -0,0 +1,50 @@
+import sbt._
+import sbt.Keys._
+
+object Build extends Build {
+
+  override lazy val settings =
+    super.settings ++ Seq(shellPrompt := { s => Project.extract(s).currentProject.id + " > " })
+
+  lazy val root = Project("Wayfinder_ROOT", file("."))
+    .aggregate(wayfinder)
+
+  lazy val wayfinder =
+    Project("wayfinder", file("wayfinder"))
+      .settings(
+        name := "wayfinder",
+        organization := "com.azavea",
+        version := "0.1-SNAPSHOT",
+        scalaVersion := "2.10.4",
+        scalacOptions ++=
+          Seq("-deprecation",
+            "-unchecked",
+            "-Yinline-warnings",
+            "-language:implicitConversions",
+            "-language:reflectiveCalls",
+            "-language:postfixOps",
+            "-language:existentials",
+            "-feature"),
+
+        libraryDependencies ++= Seq(
+          "io.argonaut" %% "argonaut" % "6.0.4",
+
+          "io.spray" % "spray-routing" % "1.2.0",
+          "io.spray" % "spray-can" % "1.2.0",
+          "io.spray" % "spray-httpx" % "1.2.0",
+          "com.typesafe.akka" %% "akka-actor" % "2.2.4",
+
+          "org.scala-lang" % "scala-compiler" % "2.10.4",
```

Comment:
I wasn't sure about that one - it was pulled into OTI (which I used as a reference).